test/test_import_time.py

```diff
@@ -7,13 +23,48 @@
 # S3 bucket (see tools/stats/print_test_stats.py)
 class TestImportTime(TestCase):
     def test_time_import_torch(self):
-        TestCase.runWithPytorchAPIUsageStderr('import torch')
+        TestCase.runWithPytorchAPIUsageStderr("import torch")
```

Comment:
Unrelated to this PR, isn't it? And if it's a new flake8 rule, can we try to enforce it consistently across the codebase?

Reply:
Yup, both this and the other PR got hit by format-on-save in VSCode. Maybe we could periodically run it on a allowlist of files in the codebase? Otherwise anyone else with it enabled will hit similar issues (especially since the formatter doesn't support formatting ranges, only entire files AFAIK)